src/phone/validators.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Validates the phone number based in the brazilian phone number format
+ * You can't use a phone number with country code
+ */
+export const isBrazilianCellPhone = (arg: string | number): boolean => {
```

Comment:
I Didn't like this name, the Brazilian cell phone is defined by the country code +55 before the number, not the format.
isBrazilianCellPhoneFormat isn't better?

Reply:
i don't agree. what do you think @caroltrindade?

Reply:
I agree...the Brazilian cell phone is defined by the format and the +55 prefix, as you only validate the format I think the function name should show this.

Reply:
ok